airbyte-integrations/connectors/source-amazon-seller-partner/metadata.yaml

```diff
@@ -2,15 +2,15 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: e55879a8-0ef8-4557-abcf-ab34c53ec460
-  dockerImageTag: 1.2.0
+  dockerImageTag: 1.3.0
```

Comment:
As far as I can tell this PR doesn't introduce any new features and just fixes an issue on our end. It also will not have any adverse effect on existing users. Following semver, this could be noted as a patch release rather than minor. Is there a special reason you went with a minor bump?

Reply:
based on versioning rules, spec changes are considered "breaking", thus minor number was incremented.